src/pages/index.scss

```diff
@@ -135,33 +135,45 @@
 .hero {
   color: $color-grey-000;
 
+  &__logo {
```

Comment:
you tried everything center your element ;)


`text-align: center;` => will center the text inside the SVG
`display: flex;` => no content, no effect... aside from displaying the image at 100% of the width
`justify-content: center;` => what content are you trying to center ? path ? 
`margin: 0 auto $mu250;` => `margin 0` auto works for blocks or inline block, but for inline elements. 


once you found the thing that works, try to remove the useless code.
this does not work on safari, and I'm surprised that it even work on chrome.

this is one right way to do it : 

`display: block;` => set the SVG as a block element
`width: $mu400;` => 
`height: $mu400;` => define width and height
`margin: 0 auto $mu250;` => center it and apply margin at the bottom

Reply:
sorry, I did not rewrite my code, it was a copy / paste and I forgot it: - /
shame on me !